tools/encodings/arabic_unique-glyphs.nam

```diff
@@ -291,27 +198,6 @@
 0xFBAF # ARABIC LETTER YEH BARREE FINAL FORM
 0xFBB0 # ARABIC LETTER YEH BARREE WITH HAMZA ABOVE ISOLATED FORM
 0xFBB1 # ARABIC LETTER YEH BARREE WITH HAMZA ABOVE FINAL FORM
-0xFBD3 # ARABIC LETTER NG ISOLATED FORM
-0xFBD4 # ARABIC LETTER NG FINAL FORM
-0xFBD5 # ARABIC LETTER NG INITIAL FORM
-0xFBD6 # ARABIC LETTER NG MEDIAL FORM
-0xFBD7 # ARABIC LETTER U ISOLATED FORM
-0xFBD8 # ARABIC LETTER U FINAL FORM
-0xFBD9 # ARABIC LETTER OE ISOLATED FORM
-0xFBDA # ARABIC LETTER OE FINAL FORM
-0xFBDB # ARABIC LETTER YU ISOLATED FORM
-0xFBDC # ARABIC LETTER YU FINAL FORM
-0xFBDD # ARABIC LETTER U WITH HAMZA ABOVE ISOLATED FORM
-0xFBDE # ARABIC LETTER VE ISOLATED FORM
-0xFBDF # ARABIC LETTER VE FINAL FORM
-0xFBE0 # ARABIC LETTER KIRGHIZ OE ISOLATED FORM
-0xFBE1 # ARABIC LETTER KIRGHIZ OE FINAL FORM
-0xFBE2 # ARABIC LETTER KIRGHIZ YU ISOLATED FORM
-0xFBE3 # ARABIC LETTER KIRGHIZ YU FINAL FORM
-0xFBE4 # ARABIC LETTER E ISOLATED FORM
-0xFBE5 # ARABIC LETTER E FINAL FORM
-0xFBE6 # ARABIC LETTER E INITIAL FORM
-0xFBE7 # ARABIC LETTER E MEDIAL FORM
 0xFBE8 # ARABIC LETTER UIGHUR KAZAKH KIRGHIZ ALEF MAKSURA INITIAL FORM
 0xFBE9 # ARABIC LETTER UIGHUR KAZAKH KIRGHIZ ALEF MAKSURA MEDIAL FORM
 0xFBFC # ARABIC LETTER FARSI YEH ISOLATED FORM
```

Comment:
I’d drop all Arabic presentation forms completely. If the font uses them in GSUB then `pyftsubset` (assuming that is what is used here) will keep them any way, if they are not used (like in Amiri) then you save file size. The only character that should be kept in this block is U+FDFA.